### PR TITLE
refactor: warning about swagger 2.0 & more string updates

### PR DESCRIPTION
**Pull request by yuqizhou77**
show warning if the file is of swagger version 2.0
image

update placeholder:
image

### CODE REVIEW

**Review by nliu-ms — DISMISSED** [review]

File: packages/vscode-extension/package.json [hunk 1]
```diff
@@ -1189,7 +1189,7 @@
         "fx-extension.developCopilotPlugin": {
           "type": "boolean",
           "default": false,
-          "markdownDescription": "Enable to develop Copilot plugin (Require to reload VSCode). Visit [How to Extend Microsoft 365 Copilot](https://learn.microsoft.com/en-us/microsoftteams/platform/copilot/how-to-extend-copilot) to learn more."
+          "markdownDescription": "Enable to develop Copilot plugin (Reload Visual Studio Code after changing this setting to take effecte). Visit [How to Extend Microsoft 365 Copilot](https://aka.ms/teamsfx-copilot-plugin) to learn more."
```
Comment (tecton) [inline packages/vscode-extension/package.json:1192]: typo: effecte -> effect

Comment (yuqizhou77) [reply packages/vscode-extension/package.json]: updated

**Review by tecton — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -250,6 +250,7 @@
   "core.copilotPlugin.validate.summary.validate.warning": "%s warning",
   "core.copilotPlugin.scaffold.summary": "We have detected the following issues for your OpenAPI specification file:\n%s",
   "core.copilotPlugin.scaffold.summary.warning.operationId": "%s Mitigation: Not required, operationId has been automatically generated and added into \"%s\" file.",
+  "core.copilotPlugin.scaffold.summary.warning.swaggerversion": "The API specification is of Swagger version 2.0. Mitigation: Not required. The content has been converted to OpenAPI 3.0 and saved in \"%s\".",
```
Comment (sffamily) [inline packages/fx-core/resource/package.nls.json:253]: is of -> is on

Comment (yuqizhou77) [reply packages/fx-core/resource/package.nls.json]: updated.

**Review by sffamily — APPROVED** [review]

**Review by nliu-ms — APPROVED** [review]

**Pull request merged**